utils/getTrustedCertificates.ts

```diff
@@ -0,0 +1,107 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as fse from 'fs-extra';
+import * as https from 'https';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import { callWithTelemetryAndErrorHandling, IActionContext } from "vscode-azureextensionui";
+import { ext } from '../extensionVariables';
+import { globAsync } from '../helpers/async';
+import { isLinux, isMac, isWindows } from '../helpers/osVersion';
+
+let _systemCertificates: (string | Buffer)[] | undefined;
+
+export async function getTrustedCertificates(): Promise<(string | Buffer)[]> {
+    // tslint:disable-next-line:no-function-expression
+    return callWithTelemetryAndErrorHandling('docker.certificates', async function (this: IActionContext): Promise<(string | Buffer)[]> {
+        this.suppressTelemetry = true;
+
+        let systemCerts = getCertificatesFromSystem();
+
+        let certificatePaths = vscode.workspace.getConfiguration('docker').get<string[] | undefined>('certificates');
+        if (certificatePaths === undefined) {
+            if (isLinux()) {
+                certificatePaths = ['/etc/ssl/certs/ca-certificates', '/etc/openssl/certs', '/etc/pki/tls/certs', '/usr/local/share/certs'];
```

Comment:
How'd you pick this list? Maybe add a comment with link to docs

Reply:
I'd like to but there is no official list, which is why I added a configuration option.

Reply:
Added this:

// These are some of the most common paths on Linux, but there is no official standard

Reply:
Talked offline. I'm fine with this since they're analagous to the "system cert store" on linux